com.avaloq.tools.ddk.xtext.builder/src/com/avaloq/tools/ddk/xtext/builder/DerivedObjectAssociations.java

```diff
@@ -0,0 +1,83 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Avaloq Evolution AG and others.

```

Comment:
We should do something about this "2016" in the templates...

Reply:
Yes. can do for this single source in this change, but the rest we need to think.